web/modules/custom/pineapple_general/pineapple_general.module

```diff
@@ -103,6 +103,7 @@ function pineapple_general_form_alter(&$form, \Drupal\Core\Form\FormStateInterfa
     case 'comment_article_comment_form':
     case 'comment_recipe_comment_form':
       $form['actions']['submit']['#value'] = t('Post');
+      $form['actions']['submit']['#attributes']['class'][] = t('btn--form-submit');
```

Comment:
As a result of this the button now has unnecessary duplicated classes, this should be done in the twig template. Not in php.

Reply:
done